### ISSUE TITLE
Restyaboard shows whitepage and keeps reloading

### ISSUE THREAD

**Issue by gcommit** [RestyaPlatform/board#4282]
I followed these instructions, but with php7.4
https://slacker.ro/2020/01/06/how-to-install-restyaboard-on-debian-buster/

When i build my restyaboard, i will get a whitepage with an orange loading bar, which keeps loading from new, every second. In the nginx logs i can see this: 

```
web_1  | 172.22.0.1 - - [07/Jul/2021:12:15:24 +0000] "GET /api/v1/oauth.json?_=1625660124279 HTTP/1.1" 404 31 "http://localhost/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36"
web_1  | 172.22.0.1 - - [07/Jul/2021:12:15:24 +0000] "GET /api/v1/oauth.json?_=1625660124305 HTTP/1.1" 404 31 "http://localhost/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36"
web_1  | 172.22.0.1 - - [07/Jul/2021:12:15:24 +0000] "GET /api/v1/oauth.json?_=1625660124328 HTTP/1.1" 404 31 "http://localhost/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36"
web_1  | 172.22.0.1 - - [07/Jul/2021:12:15:24 +0000] "GET /api/v1/oauth.json?_=1625660124354 HTTP/1.1" 404 31 "http://localhost/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36"
web_1  | 172.22.0.1 - - [07/Jul/2021:12:15:24 +0000] "GET /api/v1/oauth.json?_=1625660124379 HTTP/1.1" 404 31 "http://localhost/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36"
web_1  | 172.22.0.1 - - [07/Jul/2021:12:15:24 +0000] "GET /api/v1/oauth.json?_=1625660124413 HTTP/1.1" 404 31 "http://localhost/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36"
web_1  | 172.22.0.1 - - [07/Jul/2021:12:15:24 +0000] "GET /api/v1/oauth.json?_=1625660124441 HTTP/1.1" 404 31 "http://localhost/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36"
web_1  | 172.22.0.1 - - [07/Jul/2021:12:15:24 +0000] "GET /api/v1/oauth.json?_=1625660124472 HTTP/1.1" 404 31 "http://localhost/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36
```

Nginx is up and running, postresql, fpm as well and i get no errors during the installation

Can someone help?

**Comment by sridhar391** [comment]
@gcommit You can use the Restyaboard installation script to install Restyaboard on your debian system, 

Please attach the Nginx error log in the folder path "/var/log/nginx/error.log", it can be an issue of database connection or file permission issue

**Comment by gcommit** [comment]
The error log is empty, the db connection is successfull and i can't use your install script, since it needs manual input and i can't use it with an automated docker setup. 

Everythings needs to be installed and configured in setup.sh and Dockerfile, so when i start an "docker-compose up --build" i will get an fully installed and configured restyaboard

**Comment by gcommit** [comment]
i also changed permissions of files mentioned here: (manual install)
https://restya.com/board/docs/install-restyaboard/

**Comment by gcommit** [comment]
any suggestion?

**Comment by manakill** [comment]
Absolutely agree with the Docker suggestion. I am facing a similar issue every few months there is some kind of failure that results in this similar white screen issue. I would love to have a simple Docker image running, with storage on external hard drive setting, so that when an error was faced we could just Rebuld the Docker Container and everything would be back to working in minutes.

**Comment by sridhar391** [comment]
@gcommit Can you please send us the instructions you have followed to setup the Restyaboard in docker

**Comment by gcommit** [comment]
I already did ...

"I followed these instructions, but with php7.4
https://slacker.ro/2020/01/06/how-to-install-restyaboard-on-debian-buster/"
"i also changed permissions of files mentioned here: (manual install)
https://restya.com/board/docs/install-restyaboard/"

**Comment by gcommit** [comment]
So? Any News?

**Comment by sridhar391** [comment]
@gcommit Sorry for the late response, can you please follow the instructions from the link https://restya.com/board/docs/install-restyaboard-docker/ for installation of Restyaboard in docker

**Comment by gcommit** [comment]
No. I Can't follow these steps, since this pre-configured does not fit in my infra. Thats why i follow 
"I followed these instructions, but with php7.4
https://slacker.ro/2020/01/06/how-to-install-restyaboard-on-debian-buster/"
"i also changed permissions of files mentioned here: (manual install)
https://restya.com/board/docs/install-restyaboard/"

As is just said now for the 3rd time. 

I need help with this instructions and the whitepage resulting.

**Comment by manakill** [comment]
Sometimes I can resolve the whitepage issue by deleting cookies and temp internet files. Sometimes there is a Token left over that causes the whitepage issue

**Comment by sridhar391** [comment]
@gcommit Please set R_DEBUG as true on the config.inc.php file in line 15 on /usr/share/nginx/html/restyaboard/server/php path like https://github.com/RestyaPlatform/board/blob/dev/server/php/config.inc.php#L15 and copy the response of http://<restyaboard_url>/api/v1/oauth.json on the Network tab of Browser developer tools and paste it here.
